lightning/src/ln/channelmanager.rs

```diff
@@ -8792,20 +8797,24 @@ where
 			// one invoice for a given payment id will be paid, even if more than one is received.
 			const REQUEST_LIMIT: usize = 10;
```

Comment:
Doesn't look like the comment for this is accurate anymore. I wonder if we should keep an overall cap on requests instead of sending this many for each offer path. I'm sure there is some creative use of `take` and `cycle` that would allow us to do something like that easily.

Reply:
Updated the usage of REQUEST_LIMIT so that it now sets the limit to the number of messages created.
Also update the comment accordingly in [f32a93a](https://github.com/lightningdevkit/rust-lightning/commit/f32a93acc60d20078c42b96dbb4466969b06f5b3)
Thanks for the suggestion!